code/modules/reagents/holder.dm

```diff
@@ -369,7 +369,7 @@
 //not directly injected into the contents. It first calls touch, then the appropriate trans_to_*() or splash_mob().
 //If for some reason touch effects are bypassed (e.g. injecting stuff directly into a reagent container or person),
 //call the appropriate trans_to_*() proc.
-/datum/reagents/proc/trans_to(datum/target, amount = 1, multiplier = 1, copy = 0)
+/datum/reagents/proc/trans_to(datum/target, amount = 1, multiplier = 1, copy = 0, ignore_isinjectable = 0)
```

Comment:
That way seems kinda tricky

Reply:
Why not just update `is_injectable(...)` proc for tanks?

Reply:
Or redefine tanks `attackby`

Reply:
> Or redefine tanks `attackby`

No redefine cuz of attackby must fill beaker, and mousedrop will not fill it, i think it's better than alert with "You want to fill beaker or empty it in this tank?"